### PR TITLE
[RFR] Updated Configuration file structure

### PR DESCRIPTION
**Pull request by midays**
signed-off by midays@redhat.com

### CODE REVIEW

File: src/fixtures/ide_fixture.py [hunk 1]
```diff
@@ -6,10 +6,28 @@
 
 from src.lib.IDE.Intellij import Intellij
 from src.lib.IDE.VisualStudioCode import VisualStudioCode
-from src.lib.web import EclipseChe
```
Comment (nachandr) [inline src/fixtures/ide_fixture.py:9]: We continue to support EclipseChe. So, this needs to be retained .

File: src/fixtures/ide_fixture.py [hunk 2]
```diff
@@ -23,41 +41,32 @@ def config():
...
-    path = config["ide_paths"]["vs_code"]
+    path = vscode_config["ide_path"]
     vscode.open_application(path)
     vscode.set_default_timeout(timeout=config["timeout_in_seconds"])
     vscode.delete_config_files()
     yield vscode, config
     vscode.close_ide()
 
 
-@pytest.fixture(scope="function")
```
Comment (nachandr) [inline src/fixtures/ide_fixture.py]: This fixture for setting up  Eclipse Che needs to be retained .

**Review by nachandr — CHANGES_REQUESTED** [review]

File: src/utils/general.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import os
+import shutil
+
+
+def clear_directory_by_name(directory, name_contains):
```
Comment (nachandr) [inline src/utils/general.py]: @midays , Is this utility function being used somewhere ?

**Review by nachandr — APPROVED** [review]

File: src/fixtures/ide_fixture.py [hunk 4]
```diff
@@ -8,27 +8,46 @@
...
+    return vscode_config_data
 
 
 @pytest.fixture(scope="session")
 def config():
     """
     Fixture to configure IDE path
     """
-    with open(CONF_DIR + "config.json") as config:
+    with open(CONF_DIR + "analysis.config.json") as config:
```
Comment (sshveta) [inline src/fixtures/ide_fixture.py]: please rename to analysis_config.json

**Review by sshveta — APPROVED** [review]

**Pull request merged**